SNT/SNT.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <LangVersion>7.1</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Numerics.Tensors" Version="0.1.0-preview2-181031-3" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```

Comment:
Since I don't do unsafe C# too often, I keep forgetting -- does it not complicate using the package from applications/services that don't allow unsafe code? Is there a way to avoid that?

Reply:
Mmm I don't think I can avoid this. I need to map the unmanaged memory from TorchSharp into C# stuff; I don't think I can avoid the usage of pointers.

Reply:
Unsafe is also already used in TorchSharp when returning the pointer to data.

Reply:
So, looking at this some more -- TorchSharp.NNTensor.Data takes an IntPtr and casts it to a pointer, which makes it unsafe code. Then, your code takes the pointer and casts it back to an IntPtr for storage. If Tensor.Data instead returns the IntPtr it got from native code, you shouldn't have to handle pointers, right?

Reply:
I can definitely remove some of the unsafe in that way. However I will still need Unsafe because the Pin method accepts as input an index and therefore I need to do pointer + index to return the correct slice of memory. Actually that Unsafe dependency is used for this.

Reply:
Okay, that's good. We can get rid of the unsafe attribute on TorchSharp, then. I have a PR coming with some of those changes.